packages/react/package.json

```diff
@@ -15,20 +15,18 @@
     "cjs/",
     "umd/",
     "jsx-runtime.js",
-    "jsx-dev-runtime.js",
-    "unstable-shared-subset.js"
+    "jsx-dev-runtime.js"
   ],
   "main": "index.js",
   "exports": {
     ".": {
-      "react-server": "./unstable-shared-subset.js",
+      "react-server": "./cjs/react.shared-subset.js",
```

Comment:
This doesn't seem right. It would be `./react.shared-subset.js`, right? Because the `/npm/react.shared-subset.js` ends up there.

Reply:
Yeah you're right, I think this was copy pasta. Surprised Jest doesn't catch this. Which fixture should I run to test this?

Reply:
Tested with server-components-demo repo